.github/workflows/tests.yml

```diff
@@ -11,10 +11,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: aiken-lang/setup-aiken@v0.1.0
+      - uses: aiken-lang/setup-aiken@v1
         with:
-          version: v1.0.28-alpha
-
+          version: v1.1.0
```

Comment:
NOTE: This will only work once we have officially released.

Reply:
Thank you! I'm looking forward to it